File: Java_Practice/Java/pointer_address.java
//Print the address and value of a variable using a pointer
public class pointer_address {
    public static void main(String[] args) {
        Integer a = 10;
        Integer b = a; // reference
        System.out.println("Value: " + b);
        System.out.println("Reference (hashCode): " + System.identityHashCode(b));
    }
}

File: Java_Practice/CPP/pointer_address.cpp
//Print the address and value of a variable using a pointer
#include <iostream>
using namespace std;
int main() {
    int a = 10;
    int *p = &a;
    cout << "Value: " << *p << endl;
    cout << "Address: " << p << endl;
    return 0;
}
